lib/ddtrace/ext/test.rb

```diff
@@ -1,24 +1,8 @@
 module Datadog
   module Ext
-    # Defines constants for test tags
+    # Defines constants for test behavior
     module Test
-      TAG_ARGUMENTS = 'test.arguments'.freeze
-      TAG_FRAMEWORK = 'test.framework'.freeze
-      TAG_NAME = 'test.name'.freeze
-      TAG_SKIP_REASON = 'test.skip_reason'.freeze
-      TAG_STATUS = 'test.status'.freeze
-      TAG_SUITE = 'test.suite'.freeze
-      TAG_TRAITS = 'test.traits'.freeze
-      TAG_TYPE = 'test.type'.freeze
-
-      # TODO: is there a better place for SPAN_KIND?
-      TAG_SPAN_KIND = 'span.kind'.freeze
-
-      module Status
-        PASS = 'pass'.freeze
-        FAIL = 'fail'.freeze
-        SKIP = 'skip'.freeze
-      end
+      ENV_MODE_ENABLED = 'DD_TEST_MODE_ENABLED'.freeze
```

Comment:
Is this consistent across languages?

Reply:
No, this is an original creation as far as I'm aware.

Reply:
that's fine. Do we think this 'test mode' will require a sdk-specific env var across langs in the future, or is ruby the only lang that has a special mode/settings needed in these envs? Would it make sense to namespace `DD_RUBY_TEST_MODE_ENABLED` to avoid namespace clash in the future or is that overkill? fine either way imo.

Reply:
It's really hard to say... if I had to guess, I don't think all languages will have a need for this: it's most likely informed by how each tracer operates between production and test environments, and the accessibility to changing tracer function to suit these needs. I can see the possibility that some language doesn't have such a need, and I could potentially see a future version of the Ruby tracer that doesn't need it either.

I don't think namespacing by language makes sense, but it may be better to namespace by "product" e.g. `DD_TRACE_TEST_MODE_ENABLED`, so if that seems sensible to you, I'll make that change.

Reply:
Changed this.